src/page-response-impact/ReducingR0ImpactMetrics.tsx

```diff
@@ -22,20 +26,25 @@ const PercentValue = ({ value }: { [value: string]: string }) => {
   );
 };
 
-const ReducingR0ImpactMetrics: React.FC = () => {
-  // TODO: Replace with actual data
+interface Props {
+  reductionData: reductionCardDataType;
+}
+
+const ReducingR0ImpactMetrics: React.FC<Props> = ({ reductionData }) => {
   return (
     <ImpactMetricsContainer>
       <ImpactMetricCard
         title="Hospital beds used"
-        value={<PercentValue value={"50%"} />}
-        subtitle="12% of staff"
+        value={
+          <PercentValue
+            value={`${roundToPercent(reductionData?.hospitalBedsUsed)} %`}
+          />
+        }
         icon={hospitalIcon}
       />
       <ImpactMetricCard
         title="Additional staff able to work"
-        value={200}
-        subtitle="12% of incarcerated"
+        value={reductionData?.staffUnableToWork}
```

Comment:
Should this be `staffPopulation - staffUnableToWork` ?

Reply:
No, it is the different in peak between the original projection and current projection.

From the PRD:
`Reduction in peak hospital utilization percentage: The difference % of hospital used at peak between original and current projections
E.g., 10.4% fewer hospital beds used at peak
More staff able to work: The difference between # staff unable to work between original and current projections
E.g., 100 more staff able to work
`

Reply:
Oooh okay that makes sense now..thanks for the details

Reply:
So basically, its the number of staff that are now able able to work based on their efforts to reduce spread, compared to the original projection. Does that make sense? Am I off base here?

Reply:
I think it makes sense...I think it confused me because it's a projection and not an actual number. I understand it as:

Original projection of # of people unable to work - Current projection of # of people unable to work

and we would say `able` if original unable to work is greater than current unable to work and we would want to say `unable` if current is greater than original

Reply:
yes @daschi that's how I interpret as well, so we're all in agreement.